articles/overview/index.md

```diff
@@ -1,5 +1,10 @@
 # Welcome to Intent Architect
 
+>[!NOTE]
+>With Intent Architect we want you to stop focusing on the nitty-gritty aspects of Software development and really become more productive by completing new features that you need to build to really add value to your business.
```

Comment:
I don't think the term "nitty-gritty" is correct here. I think there will still be plenty nitty-gritty details in the unavoidable custom business code, but what Intent Architect eliminates is more of the mundane, plumbing or infrastructure work.

Reply:
I tried thinking of an expressed that was better suited than "nitty gritty" so I agree with you.